packages/test/test-drivers/src/odspTestDriver.ts

```diff
@@ -29,66 +29,100 @@ const passwordTokenConfig = (username, password): OdspTokenConfig => ({
     password,
 });
 
-export interface IOdspConfig extends IClientConfig {
+export interface IOdspTestLoginInfo {
     server: string;
-    driveId: string;
     username: string;
+    password: string;
+}
+export interface IOdspTestDriverConfig extends IClientConfig, IOdspTestLoginInfo {
+    driveId: string;
     directory: string;
 }
 
 export class OdspTestDriver implements ITestDriver {
     public static createFromEnv() {
-        const config = JSON.parse(fs.readFileSync("./odspConfig.json", "utf-8")) as IOdspConfig;
-        const password = process.env.fluid__odsp__password;
-        assert(password, "Missing password");
+        const config: IOdspTestDriverConfig = {
+            username: process.env.testuser!,
```

Comment:
this is fine for now, but we should unify the format here with the service load test

Reply:
Yep.   just hacking it for now to make something working.